core/src/main/resources/schema/postgres/postgres-create-schema.sql

```diff
@@ -23,11 +23,13 @@ CREATE UNIQUE INDEX event_journal_ordering_idx ON public.event_journal(ordering)
 
 CREATE TABLE IF NOT EXISTS public.event_tag(
     event_id BIGINT,
+    persistence_id VARCHAR(255),
+    sequence_number BIGINT,
     tag VARCHAR(256),
-    PRIMARY KEY(event_id, tag),
+    PRIMARY KEY(persistence_id, sequence_number, tag),
     CONSTRAINT fk_event_journal
-      FOREIGN KEY(event_id)
-      REFERENCES event_journal(ordering)
+      FOREIGN KEY(persistence_id, sequence_number)
+      REFERENCES event_journal(persistence_id, sequence_number)
       ON DELETE CASCADE
```

Comment:
I have confirmed this approach. It can first add the column and then, towards the end of the migration, finalize the changes to the table's primary key and foreign key.